python/tank/bootstrap/resolver.py

```diff
@@ -255,31 +264,48 @@ def resolve_shotgun_configuration(
             # resolve primary and user config
             primary_config = None
             user_config = None
+            secondary_primary_config = None
+            secondary_user_config = None
```

Comment:
It looks like the only time these `secondary_` variables are set, there are log statements about being `Site-level` matches. Perhaps a `_fallback` suffix then? `primary_config_fallback` and `user_config_fallback`?